src/mz/src/api.rs

```diff
@@ -259,7 +228,7 @@ pub async fn list_cloud_providers(
         .await
 }
 
-pub async fn get_provider_by_region_name(
+pub async fn get_cloud_provider(
```

Comment:
I can't comment on the `CloudProvider` struct above, but the response from the global api will include an additional field in each CloudProvider object called `region_api_url` that needs to be used instead of `api_url` (see https://github.com/MaterializeInc/cloud/pull/6981)

and we need to update wherever `{provider.api_url}/api/region` is used to switch that to `{provider.region_api_url}/api/region`

I used a new field rather than re-writing the existing one so that I could roll out the global api change before the CLI & Web Console were updated. Eventually I'll remove the `api_url` field

Reply:
Sorry - based on PR feedback I just switched the new field name to be `url` instead of `region_api_url` in https://github.com/MaterializeInc/cloud/pull/6981